kahuna/public/js/components/gr-chips/gr-chips.html

```diff
@@ -0,0 +1,18 @@
+<div class="gr-chips__wrapper"
+     ng:click="$grChipsCtrl.wrapperClicked($event)">
```

Comment:
Why `$grChipsCtrl` rather than `ctrl` (which seems to be preferred in this project)?